README.md

```diff
@@ -12,4 +12,5 @@ Containers
 - mixer-ci
 - Keystone
 - MariaDB
+- stacks
```

Comment:
Ah this is not correct because these are the container name ie what is used for ```clearlinux:$name``` and stacks is probably a prefix so stacks-dlaas-mkl for instance would be the name.